src/commitment_scheme/kzg10/key.rs

```diff
@@ -163,7 +174,7 @@ impl CommitKey {
     /// However we note that the quotient polynomial is invariant under the value f(z)
     /// ie. only the remainder changes. We can therefore compute the witness as f(x) / x - z
     /// and only use the remainder term f(z) during verification.
-    pub fn compute_single_witness(&self, polynomial: &Polynomial, point: &BlsScalar) -> Polynomial {
+    fn compute_single_witness(&self, polynomial: &Polynomial, point: &BlsScalar) -> Polynomial {
```

Comment:
Apparently this function is never used now. Should be removed?

Reply:
Check https://github.com/dusk-network/plonk/pull/440#issuecomment-792723104

Reply:
So, you should fix that in this PR. To be clear, it's okay to have a PR to clean up warnings, however here the warnings are due to the change of visibility: if a change the visibility of a function, and this function is never used, I should remove the function in the same PR.

Reply:
So, you should fix that in this PR. To be clear, it's okay to have a PR to clean up (generic) warnings, however here the warnings are due to the change of visibility that you're doing in this very PR: therefore if you change the visibility of a function, and this function is never used, I should remove the function in the same PR.